File: ClassesAndObjects.cpp
//ClassesAndObjects
#include<iostream>
using namespace std;
class rectangle
{
    private:
        float len, bre;
        
    public:
        static int count;
        void setdata(float l, float b)
        {
            len=l;
            bre=b;
        }
        void getdata()
        {
            cin>>len>>bre;
        }
        void displaydata()
        {
            count++;
            cout<<count<<" -> "<<len<<" "<<bre<<endl;
        }
        void area()
        {
            float area;
            area=len*bre;
            cout<<area<<endl;
        }
        void peri();
        static void resetcount()
        {
            count=0;
        }
};
int rectangle::count=0;
void rectangle::peri()
{
    float peri;
    peri=2*(len+bre);
    cout<<peri<<endl;
}
int main()
{
    rectangle r1, r2;
    rectangle  *ptr=&r2;
    r1.setdata(33.66, 66.33);
    r1.displaydata();
    r1.area();
    r1.peri();
    ptr->getdata();
    ptr->displaydata();
    ptr->area();
    ptr->peri();
    cout<<rectangle::count<<endl;
    rectangle::resetcount();
    return 0;
}

File: ClassesAndObjects.java
//ClassesAndObjects
import java.util.Scanner;
class rectangle
{
    double len, bre;

    void setdata(double l, double b)
    {
        len=l;
        bre=b;
    }
    void getdata()
    {
        Scanner Sc=new Scanner(System.in);
        len=Sc.nextDouble();
        bre=Sc.nextDouble();
        Sc.close();
    }
    void displaydata()
    {
        System.out.println(len+" "+bre);
    }
    void area()
    {
        double area;
        area=len*bre;
        System.out.println(area);
    }
    void peri()
    {
        double peri;
        peri=(len+bre)*2;
        System.out.println(peri);
    }
}
public class ClassesAndObjects
{
    public static void main(String[] args)
    {
        rectangle r1=new rectangle();
        rectangle r2=new rectangle();
        r1.setdata(22.44, 33.66);
        r1.displaydata();
        r1.area();
        r1.peri();
        r2.getdata();
        r2.displaydata();
        r2.area();
        r2.peri();
    }
}